doc/examples/edges/plot_circular_elliptical_hough_transform.py

```diff
@@ -80,9 +80,11 @@
 # -------------------
 #
 # The algorithm takes two different points belonging to the ellipse. It
-# assumes that it is the main axis. A loop on all the other points determines
-# how much an ellipse passes to them. A good match corresponds to high
-# accumulator values.
+# assumes that it is the major axis. A loop on all the other points determines
```

Comment:
```suggestion
# assumes that it is the major axis. A loop on all the other pairs of points determines
```

Reply:
Also, in "... it is the major axis," 'it' refers to the 'two different points' of the previous sentence... I guess the meaning is obvious(?) but the grammar is not. I would suggest writing: "It assumes that these two points define the major axis" (first 'it' being 'the algorithm'). Maybe replacing 'define' with 'form' or another verb?

Reply:
So I've re-read the implementation in https://github.com/scikit-image/scikit-image/blob/main/skimage/transform/_hough_transform.pyx as it's been a while since I went through it, and it's not *pairs* of points at that stage.

The two points form the major axis, and from them the algorithm computes the center of the ellipse. Each other (single) point can then be used to compute the parameters of the potential ellipse (fig. 1 from the [reference paper](https://citeseerx.ist.psu.edu/viewdoc/download;jsessionid=61678F4E21E15924FD49D66A179409C6?doi=10.1.1.1.8792&rep=rep1&type=pdf) makes it more clear).

I agree with the other remark.